python/perspective/perspective/widget/widget.py

```diff
@@ -201,6 +203,8 @@ class PerspectiveWidget(DOMWidget, PerspectiveViewer):
     _view_module = Unicode("@finos/perspective-jupyterlab").tag(sync=True)
     _view_module_version = Unicode("~{}".format(__version__)).tag(sync=True)
 
+    _enable_experimental_html_export = False
```

Comment:
It's ok to make this non-default but let's make it a real API. My proposal is two-fold:
1. Environment variable `PSP_JUPYTER_HTML_EXPORT` whose presence enables this behavior, so `nbconvert` + `nbclient` can be called from the command line.
2. A `import set_jupyter_html_export from perspectve` method which sets this env variable for the current process, so the behavior can be defined in the notebook itself (alternatively they could just set the env var but I think this is cleaner and parallels e.g. `set_num_threads()` global config APIs we already expose).

Reply:
Left is running jupyterlab with `PSP_JUPYTER_HTML_EXPORT=1`, right is running without the environment variable set. It would have been cool including a cell with the value of `os.environ.get("PSP_JUPYTER_HTML_EXPORT", None)` but I didn't think of it in time.

<img width="1452" alt="Screenshot 2023-11-14 at 3 49 59 PM" src="https://github.com/finos/perspective/assets/667901/01a7f462-c9a4-4181-aee0-9bfa3083a790">